charts/kueue/README.md

```diff
@@ -0,0 +1,56 @@
+# Kueue's helm chart
+
+## Table of Contents
+
+<!-- toc -->
+- [Installation](#installation)
+  - [Prerequisites](#prerequisites)
+  - [Installing the chart](#installing-the-chart)
+    - [Install chart using Helm v3.0+](#install-chart-using-helm-v30)
+    - [Verify that controller pods are running properly.](#verify-that-controller-pods-are-running-properly)
+  - [Configuration](#configuration)
+<!-- /toc -->
+
+## Installation
```

Comment:
Table of contents should be a lvl higher than the rest of the items in the document (2 # vs 3# after) and so on